src/dune/dune_env.mli

```diff
@@ -24,6 +24,12 @@ module Stanza : sig
     val decode : t Dune_lang.Decoder.t
   end
 
+  module Coq : sig
+
+    type t = Ordered_set_lang.Unexpanded.t
```

Comment:
We don't have type aliases for other env flag fields, so let's keep it consistent and not introduce them here.